src/main/scala/modbat/mbt/Main.scala

```
@@ -28,13 +29,11 @@ object Main {
 	  if (!remainingArgs.hasNext) {
 	    Log.error(c.header)
 	    Log.error("Model class argument missing. Try --help.")
-	    return 1
 	  }
 	  modelClassName = remainingArgs.next
 	  if (remainingArgs.hasNext) {
 	    Log.error("Extra arguments starting at \"" + remainingArgs.next() +
 		      "\" are not supported.")
-	    return 1
```

You need to throw an exception here (suggestion: IllegalArgumentException).